### DISCUSSION TITLE
Questions about "Mind map file was changed by another program..."

### DISCUSSION THREAD

**Discussion by sgjohnson1981** [freeplane/freeplane#1360] [Help]
I was interested in Freeplane add-on development, but the, at least seemingly, less complicated freeplane-python-io might do the trick. All I wanted to do, at least for now, is scan nodes for particular attributes to read/modify them according to what's going on in a CalDAV server. So naturally, I had some questions about the feature in this discussion's title. So following from the discussion below, am I correct in assuming if I deselect "Monitor external mind map file changes" it will completely ignore external changes, rather than not bug you with the message? What happens if you make changes both externally and internally and try saving it? If I keep that setting selected and choose to "Don't ask again" what precisely is that setting: per map, per app, or per app session?
https://github.com/freeplane/freeplane/discussions/484

**Comment by dpolivaev** [comment]
Per app

**Comment by dpolivaev** [comment]
more precisely, per user.
